tests/Propel/Tests/Generator/Platform/PlatformTestBase.php

```diff
@@ -9,16 +9,20 @@
 namespace Propel\Tests\Generator\Platform;
 
 use Propel\Generator\Builder\Util\SchemaReader;
-use Propel\Tests\TestCase;
+use Propel\Runtime\Propel;
+use Propel\Tests\TestCaseFixturesDatabase;
 
 /**
  * Base class for all Platform tests
  */
-abstract class PlatformTestBase extends TestCase
+abstract class PlatformTestBase extends TestCaseFixturesDatabase
 {
     protected function getDatabaseFromSchema($schema)
     {
-        $xtad = new SchemaReader($this->getPlatform());
+        $platform = $this->getPlatform();
+        require_once __DIR__ . '/../../../../Fixtures/platform/build/conf/platform-conf.php';
+        $platform->setConnection(Propel::getConnection('platform'));
+        $xtad = new SchemaReader($platform);
```

Comment:
With the changes above, this is not necessary anymore. Please undo it.
If there was a need to add a connection, we should use an existing one (like `tests/Fixtures/bookstore/build/conf/bookstore-conf.php`)

Reply:
Done